.github/dependabot.yml

```diff
@@ -6,19 +6,38 @@ updates:
     directory: "/"
     schedule:
       interval: "monthly"
+    ignore:
+      - dependency-name: "*"
+        update-types: ["version-update:semver-major"]
+        
   - package-ecosystem: "github-actions"
     directory: "/"
     schedule: 
       interval: monthly
+    ignore:

```

Comment:
Usually major version changes in github-actions dont have breaking changes. For example from v2 to v3, all it did was update the node runtime to v16

Reply:
Thanks @gaby - we did find breaking changes, particularly on the NPM/React side of things. Anything major we will manually review and implement :)